python/cudf/cudf/core/tokenize_vocabulary.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+from __future__ import annotations
+
+import cudf
+from cudf._lib.nvtext.tokenize import (
+    Tokenize_Vocabulary as cpp_tokenize_vocabulary,
+    tokenize_with_vocabulary as cpp_tokenize_with_vocabulary,
+)
+
+
+class TokenizeVocabulary:
+    """
+    Parameters
+    ----------
+    vocabulary : str
+        Strings column of vocabulary terms
+    Returns
+    -------
+    TokenizeVocabulary
+    """
```

Comment:
Fix up this docstring to adhere to https://numpydoc.readthedocs.io/en/latest/format.html. Needs a one-line summary and a blank line between sections.

```suggestion
    """
    A vocabulary object used to tokenize input text.
    
    Parameters
    ----------
    vocabulary : str
        Strings column of vocabulary terms

    Returns
    -------
    TokenizeVocabulary
    """
```